src/e3/decorator.py

```diff
@@ -51,72 +49,3 @@ def empty_func(*args: Any, **kargs: Any) -> None:
         del args, kargs
 
     return empty_func
-
-
-class memoize:
```

Comment:
We're not yet ready to drop memoize, still in use at AdaCore

Reply:
Only two references I think

Reply:
Almost remove but cannot wait in order to move to pyproject.toml. I have removed for the moment the commit